src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -372,20 +372,27 @@ function loadFeedbackResponseComments(user, courseId, fsName, fsIndx, sender) {
     var fsNameForUrl = encodeURIComponent(fsName);
     var url = "/page/instructorFeedbackResponseCommentsLoad?user=" + user + "&courseid=" + courseId + "&fsname=" + fsNameForUrl + "&fsindex=" + fsIndx;
     $(sender).find('div[class^="placeholder-img-loading"]').html("<img src='/images/ajax-loader.gif'/>");
-    panelBody.load(url, function( response, status, xhr ) {
-        if (status == "success") {
-            panelBody.removeClass('hidden');
-            updateBadgeForPendingComments(panelBody.children(":first").text());
-            panelBody.children(":first").remove();
-            registerResponseCommentsEvent();
-            registerCheckboxEventForVisibilityOptions();
-            enableHoverToDisplayEditOptions();
-      } else {
-          panelBody.find('div[class^="placeholder-error-msg"]').removeClass('hidden');
-          panelBody.removeClass('hidden');
-      }
-      $(sender).find('div[class^="placeholder-img-loading"]').html("");
-    });
+   
+    if(panelBody.hasClass('hidden')){
```

Comment:
Do also remember to address the comments for students panels in the other pages mentioned in the issue, namely `instructorSearch.jsp`, `instructorStudentRecords.jsp` and `studentComments.jsp`.